pandas/io/json/_normalize.py

```diff
@@ -283,6 +371,14 @@ def _pull_records(js: Dict[str, Any], spec: Union[List, str]) -> List:
     else:
         raise NotImplementedError
 
+    if all(
```

Comment:
can you move the comment here and explain what this is checking

Reply:
Have updated comment to be more explicit about what it is checking